api/BroadcastChannel.json

```
@@ -39,7 +39,14 @@
             "version_added": "41"
           },
           "safari": {
-            "version_added": false
+            "version_added": "preview",
+            "flags": [
+              {
+                "name": "BroadcastChannel API",
+                "type": "preference",
+                "value_to_set": "enabled"
```

Conventionally, we've omitted `value_to_set` for Safari prefs since there's no enabled string in the UI and it seems a bit noisy to include the check mark. This change would need to be applied throughout.

```suggestion
                "type": "preference"
```